src/core_modules/capture-core/components/Widget/Widget.component.js

```diff
@@ -1,144 +1,21 @@
-            borderBottomRightRadius: 0,
-        },
-    },
-    header: {
-        display: 'flex',
-        alignItems: 'center',
-        justifyContent: 'space-between',
-        padding: spacersNum.dp16,
-        fontWeight: 500,
-        fontSize: 16,
-        color: colors.grey800,
-    },
-    children: {
-        backgroundColor: colors.white,
-        borderTopLeftRadius: 0,
-        borderTopRightRadius: 0,
-        borderBottomLeftRadius: 3,
-        borderBottomRightRadius: 3,
-        borderStyle: 'solid',
-        borderColor: colors.grey400,
-        borderWidth: 1,
-        borderTopWidth: 0,
-        overflow: 'hidden',
-        '&.open': {
-            animation: 'slidein 200ms normal forwards ease-in-out',
-            transformOrigin: '50% 0%',
-        },
-        '&.close': {
-            animation: 'slideout 200ms normal forwards ease-in-out',
-            transformOrigin: '100% 0%',
-        },
-    },
-    toggleButton: {
-        margin: spacersNum.dp4,
-        color: colors.grey600,
-        '&.open': {
-            animation: 'flipOpen 200ms normal forwards linear',
-        },
-        '&.close': {
-            animation: 'flipClose 200ms normal forwards linear',
-        },
-        '&.closeinit': {
-            transform: 'rotateX(180deg)',
-        },
-    },
-    '@keyframes slidein': {
-        from: { transform: 'scaleY(0)' },
-        to: { transform: 'scaleY(1)' },
-    },
-    '@keyframes slideout': {
-        from: { transform: 'scaleY(1)' },
-        to: { transform: 'scaleY(0)' },
-    },
-    '@keyframes flipOpen': {
-        from: { transform: 'rotateX(180deg)' },
-        to: { transform: 'rotateX(0)' },
-    },
-    '@keyframes flipClose': {
-        from: { transform: 'rotateX(0)' },
-        to: { transform: 'rotateX(180deg)' },
-    },
-});
-
-export const WidgetPlain = ({ header, open, onOpen, onClose, children, classes }: Props) => {
-    const [childrenVisible, setChildrenVisibility] = useState(open); // controls whether children are rendered to the DOM
-    const [animationsReady, setAnimationsReadyStatus] = useState(false);
-    const [postEffectOpen, setPostEffectOpenStatus] = useState(open);
-    const hideChildrenTimeoutRef = useRef(null);
-    const initialRenderRef = useRef(true);
-
-    useEffect(() => {
-        if (initialRenderRef.current) {
-            initialRenderRef.current = false;
-            return;
-        }
-
-        if (!animationsReady) {
-            setAnimationsReadyStatus(true);
-        }
-
-        setPostEffectOpenStatus(open);
-
-        clearTimeout(hideChildrenTimeoutRef.current);
-        if (open) {
-            setChildrenVisibility(true);
-        } else {
-            hideChildrenTimeoutRef.current = setTimeout(() => {
-                setChildrenVisibility(false);
-            }, 200);
-        }
-    }, [open, animationsReady]);
-
-    return (
+export const Widget = ({ collapsible = true, ...passOnProps }: Props) => (
```

Comment:
Hey @EirikHaugstulen, 

I am a bit confused about the usage of the prop `collapsible`. I think it can be misleading to have a missing prop default value set to `true`. How about setting it to `false`? You will just have to add it as a prop to the components that already use `Widget` and it will work. WDYT?

Reply:
I can see how this might be misleading, and I'm not sure of what is the best solution, but I can at least explain what I think is the thought-process behind this:

At least my own mental model on using the widget component is that it should be collapsible. If it's not collapsible, it's just a basic tile or div with a header. This collapsible-prop would therefore be a way to override this if necessary. (To increase component reusability). I also think that based on the design sketches of the enrollment dashboard, we use the collapsible widget a whole lot more than the non-collapsible widget and therefore this would be an edge case. (error & warning mostly)

Highly open for refactoring this if you still think it is better to default the prop to false. ☺️
Thanks!

Reply:
How about using a prop named `noncollapsible`? If `noncollapsible` is undefined or false then the default behavior will apply and the widget will be collapsible. For the edge cases when we need to override de default behavior we can pass the prop `noncollapsible` to the Widget. Do you think it will be less confusing?

Reply:
Yep, that sounds like a good idea. 
I'll implement the changes suggested!

Reply:
Great discussion here @EirikHaugstulen @simonadomnisoru (I was the one introducing `collapsible = true` before @EirikHaugstulen inherited this task). Naming variables are really hard sometimes 🙂 Gave it some thought and I agree with you @simonadomnisoru. I think what they say here makes sense: https://www.serendipidata.com/posts/naming-guidelines-for-boolean-variables. TLDR; Avoid negative names. But ALWAYS choose names that enable setting “false” by default.